cmd/oci-runtime-tool/generate.go

```diff
@@ -93,6 +93,7 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "uidmappings", Usage: "add UIDMappings e.g HostID:ContainerID:Size"},
 	cli.StringFlag{Name: "user", Usage: "user namespace"},
 	cli.StringFlag{Name: "uts", Usage: "uts namespace"},
+	cli.StringSliceFlag{Name: "device", Usage: "specifies a device which must available in the container"},
```

Comment:
Please put it before disable-oom-kill option in alphabetical order and don't forget to add bash completion, manpage of it.